Data/Sys/GameSettings/GV4E69.ini

```diff
@@ -0,0 +1,8 @@
+# GV4E69 - MVP Baseball 2005
+
+[OnFrame]
+$Fix 2D Rendering
+0x80319214:dword:0x48113250
+
+[OnFrame_Enabled]
```

Comment:
You shouldn't use `[OnFrame_Enabled]` in a default game INI, since this makes it impossible for the user to disable the patch and makes the behavior of the patches tab in the game properties a bit confusing. JMC and I discussed a day or two ago that this is something we should do about in Dolphin's code (but what exactly?)